api4/openGraph_test.go

```diff
@@ -61,19 +61,14 @@ func TestGetOpenGraphMetadata(t *testing.T) {
 
 		openGraph, resp := Client.OpenGraph(ts.URL + data["path"].(string))
 		CheckNoError(t, resp)
-		if strings.Compare(openGraph["title"], data["title"].(string)) != 0 {
-			t.Fatal(fmt.Sprintf(
-				"OG data title mismatch for path \"%s\". Expected title: \"%s\". Actual title: \"%s\"",
-				data["path"].(string), data["title"].(string), openGraph["title"],
-			))
-		}
 
-		if ogDataCacheMissCount != data["cacheMissCount"].(int) {
-			t.Fatal(fmt.Sprintf(
-				"Cache miss count didn't match. Expected value %d. Actual value %d.",
-				data["cacheMissCount"].(int), ogDataCacheMissCount,
-			))
-		}
+		require.Equalf(t, openGraph["title"], data["title"].(string),
+			"OG data title mismatch for path \"%s\". Expected title: \"%s\". Actual title: \"%s\"",
```

Comment:
Same here. You don't need to log them.

Reply:
It seams like you did not fix this

Reply:
This one is now updated in the same way as those in `license_test` to rely on the verbosity of `testify`; thanks!